src/hmem_rocr.c

```diff
@@ -1054,6 +1063,100 @@ int rocr_dev_reg_copy_from_hmem(uint64_t handle, void *dest, const void *src,
 	return FI_SUCCESS;
 }
 
+bool rocr_is_dmabuf_supported(void)
```

Comment:
Is this function intended to be used anywhere else? If not, maybe define it as static. If yes, consider moving the `checked` logic from the caller side to here to avoid duplicated checking.

Reply:
that makes sense and I made it static, the `hmem_cuda` counterpart also have this function but non-static, I wonder if it had a specific reason there?